src/llmcompressor/modifiers/quantization/gptq/base.py

```diff
@@ -206,26 +205,24 @@ def on_initialize(self, state: "State", **kwargs) -> bool:
         if not self.quantize:
             raise ValueError("To use the GPTQModifier, quantization must be enabled.")
 
+        # prepare module names
+        self._module_names = {m: name for name, m in state.model.named_modules()}
```

Comment:
Nit:
`self._module_names = {module: name for name, module in state.model.named_modules()}`

Reply:
I shortened the name so it would fit on one line and be easier to read 🙃